README.md

```diff
@@ -8,74 +8,113 @@ Head onto our [Discord](https://discord.gg/HM8hhbGSjd) for extended support on b
-         1. Run `apt-get install libglew-dev libglfw3 libglm-dev libasio-dev`
-         2. Run `apt-get install liblua5.2-dev libopenal-dev  libbullet-dev`
-         3. Run `apt-get install libprotobuf-dev protobuf-compiler libspdlog-dev`
-      1. If you are on Arch/etc. (something with `pacman`):
-         1. Run `pacman -S glew glfw-x11 glm asio lua52 openal bullet protobuf spdlog`
-   1. `cmake ..` in the build directory
-   1. `make tec` in the build directory
-
-#### Mac OS
-   1. `cmake -DCMAKE_TOOLCHAIN_FILE=**VCPKG_DIR**/scripts/buildsystems/vcpkg.cmake ..` in the build directory
-   1. `make` in the build directory
-
-### Part 3 (Unit Tests)
-To generate the unit tests, follow the same instructions from before, but set to true the flag `BUILD_TESTS_TEC`
-
-## Clang Format
+TEC requires cmake 3.19, a c++17 compatible compiler, and a few libraries GLFW3, GLM, ASIO, Protobuf, GLEW, Lua, Bullet, Dear ImGui, sol3, Spdlog and OpenAL.
+
+The libraries are automatically installed when using CMake and VCPKG [manifest mode](https://vcpkg.readthedocs.io/en/latest/users/manifests/).
+
+# Quick Start
+## OS Specific first steps
+### Windows
+*Potentially* Download and install oalinst.zip [OpenAL installer](http://openal.org/downloads/) and install it.
+### MacOS
+Prior to 11.0.1, run (NOT TESTED): `sudo installer -pkg /Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg -target /` 
+
+If `bootstrap-vcpkg` fails see [here](https://donatstudios.com/MojaveMissingHeaderFiles) for more help
+### Linux
+Install the following. This is a pretty extensive list and may be more than needed
+
+`build-essentials pkg-config tar curl zip unzip gdb libgl1-mesa-dev xorg-dev libglu1-mesa-dev libxinerama-dev libxcursor-dev`
+
+## Build Steps
+1. Install [CMake](https://cmake.org/download/)
+2. Install [VCPK](git clone https://github.com/Microsoft/vcpkg.git) - `git clone https://github.com/Microsoft/vcpkg.git`
+3. `VCPKG_ROOT` must be set a an environment variable, to wherever VCPKG was cloned to
+4. Run `$(VCPKG_ROOT)/bootstrap-vcpkg.[bat|sh]` to setup vcpkg. Use the correct extension for your OS.
+   1. [OPTIONAL] `$(VCPKG_ROOT)/./vcpkg integrate install`
+5. Open the folder in an editor OR see step 5
+   1. [Visual Studio](https://docs.microsoft.com/en-us/cpp/build/cmake-presets-vs?view=msvc-170)
+   2. [CLion](https://www.jetbrains.com/help/clion/cmake-presets.html)
+6. Run CMake
+   1. CLI
+      1. `cmake --preset=ninja-multi-vcpkg`
+      2. `cmake --build --preset=BUILD_PRESET` see [presets](#presets)
+   2. GUI - This will only configure the project. CLI steps or Open in an editor are more useful.
+      1. Set the source folder
+      2. Run Configure
+         1. Pick the Ninja Multi-config generator
+         2. Select Specify toolchain file for cross compiling
+         3. The file input should be filled in, but if not point it to `$(VCPKG_ROOT)/scripts/buildsystems/vcpkg.cmake`
+      3. Run Generate
```

Comment:
If cmake-gui was used, then here we need to specify opening the SLN file in the C++ IDE and building the solution.